conf/web2py

```diff
@@ -9,38 +9,29 @@ SRC=/usr/local/src
 W2PPASS=turnkey
 W2PROOT=/var/www/web2py
 WSGI=$W2PROOT/wsgihandler.py
+RELEASES='https://api.github.com/repos/web2py/web2py/releases/latest'
 
 # clone web2py
 cd /var/www
-RELEASES="$(/usr/local/bin/gh_releases web2py/web2py)"
-VER=$(echo "${RELEASES}" | sort --version-sort | tail -1)
-git clone --recursive --branch $VER https://github.com/web2py/web2py
 
-# Install GitPython in virtualenv to workaround turnkey-pylib
-# git.py conflict
-virtualenv -p python2 $W2PROOT/venv2
-$W2PROOT/venv2/bin/pip install GitPython
+TAG="$(
+    curl "$RELEASES" -s \
+    | grep --color=never '"tag_name":' \
+    | grep -o '[0-9][^"]\+'
+)"
```

Comment:
This is pulling in "release" `2.19.1` : https://github.com/web2py/web2py/releases/tag/2.19.1 (from Mar 22, 2020).

But latest is `v2.26.1`: https://github.com/web2py/web2py/releases/tag/v2.26.1 (from Oct 30, 2023)

Was using that much older version intentional?

FWIW it looks like they used to do "releases" but `2.19.1` was the last one. It looks like they've switched to just using tags (and have added a leading 'v').

FWIW, this gives the correct tag:

```
gh_releases web2py/web2py | sort -V | tail -1
```

I.e.:
```
root@tkldev8 ~# TAG=$(gh_releases web2py/web2py | sort -V | tail -1)
[WARNING] GITHUB_USER not set! GITHUB_USER_TOKEN not set! Authentication won't be used.
Fetching releases from github for 'web2py/web2py'... Done!
root@tkldev8 ~# echo $TAG
v2.26.1
```

Obviously you could tweak your code to achieve the same ends, but `gh_releases` (in common) takes care of it too.